test/automation/src/sql/notebook.ts

```diff
@@ -295,6 +315,34 @@ export class NotebookToolbar {
 	async waitForNotTrustedIcon(): Promise<void> {
 		await this.code.waitForElement(NotebookToolbar.notTrustedButtonSelector);
 	}
+
+	async collapseCells(): Promise<void> {
+		let buttons: IElement[] = await this.code.waitForElements(NotebookToolbar.toolbarButtonSelector, false);
+		buttons.forEach(async button => {
+			if (button.className.includes('icon-collapse-cells')) {
+				await this.code.waitAndClick(NotebookToolbar.collapseCellsButtonSelector);
+				return;
```

Comment:
Are you intending to exit the loop early with this? forEach doesn't allow that - you need to use an actual foreach loop if you want to do that. (same on line 334)

Reply:
Could just use find instead to find the button - that seems to be more aligned with what the logic is trying to do anyways.